hack/generate-yamls.sh

```diff
@@ -74,8 +74,10 @@ readonly KO_YAML_FLAGS="${KO_YAML_FLAGS} ${KO_FLAGS} --strict"
 
 if [[ -n "${TAG}" ]]; then
   LABEL_YAML_CMD=(sed -e "s|serving.knative.dev/release: devel|serving.knative.dev/release: \"${TAG}\"|")
+  NET_ISTIO_YAML=https://github.com/knative/net-istio/releases/download/${TAG}/release.yaml
 else
   LABEL_YAML_CMD=(cat)
+  NET_ISTIO_YAML=https://storage.googleapis.com/knative-nightly/net-istio/latest/release.yaml
```

Comment:
This link is broken?  How do I make 'latest' available to download?

Reply:
I just tried `wget https://storage.googleapis.com/knative-nightly/net-istio/latest/release.yaml`. It worked for me.